pyfr/integrators/dual/pseudo/multip.py

```diff
@@ -61,6 +61,10 @@ def __init__(self, backend, systemcls, rallocs, mesh, initsoln, cfg,
             if l == self._order:
                 bases = [cc, pc]
             else:
+                if pn.startswith('optrk'):
```

Comment:
This smells funny.  Add a property to the pseudo steppers which describes the attribute of the scheme you are looking for rather than relying on a naming convention.